internal/service/organization/resource_organization_test.go

```diff
@@ -143,6 +143,24 @@ func TestAccConfigRSOrganization_Settings(t *testing.T) {
 	})
 }
 
+func TestAccOrganizationCreate_Errors(t *testing.T) {
+	var (
+		roleName    = "ORG_OWNER"
+		unknownUser = "65def6160f722a1507105aaa"
+	)
+	acc.SkipTestForCI(t) // test will fail in CI since API_KEY_MUST_BE_ASSOCIATED_WITH_PAYING_ORG is returned
```

Comment:
is it reasonable easy to enable this test in CI?

Reply:
Captured in this ticket: https://jira.mongodb.org/browse/CLOUDP-264042